docs/OnnxReleases.md

```diff
@@ -62,13 +62,6 @@ The ONNX project, going forward, will plan to release roughly on a four month ca
 * Test the PyPI package installation with different combinations of various Python versions, Protobuf versions and platforms.
   * Python versions : Applicable python versions for the release.
   * Protobuf versions : Latest protobuf version at the time of the release + protobuf version used for previous release
-  * Utilize the following matrix to check:
```

Comment:
Wondering what this is/was?

Reply:
Just feel no need to manually update the Python version here every time. The sentence above has expressed the same meaning.

Reply:
I see. Ok. But isn't the testing of the PyPI package installation automated (the set of combinations that are tested)? Is this done manually?

Reply:
Yes, you are right. That's another reason. Now release CI pipelines in onnx repo has covered every case and so the release manager doesn't need to test every of them manually. Ideally I would say the release manager can just test the wheel in their local for 1-2 environments, which should be sufficient. Thus I don't want the document here have explicit matrix for testing.